server/monitor/src/main/java/org/apache/accumulo/monitor/servlets/TServersServlet.java

```diff
@@ -314,13 +314,22 @@ static void doTserverList(HttpServletRequest req, StringBuilder sb, List<TabletS
     long now = System.currentTimeMillis();
 
     double avgLastContact = 0.;
+    double avgResponseTime = 0.;
+    int count = 0;
     for (TabletServerStatus status : tservers) {
+      count++;
       avgLastContact += (now - status.lastContact);
+      avgResponseTime += status.responseTime;
```

Comment:
Did you give any thought to including other statistics than just avg? I am thinking about the case where there are one or a few outliers which significantly skew the mean. p95/p99 are nice to give some context, but admittedly require a little more work to compute.